queries/snakemake/injections.scm

```diff
@@ -0,0 +1,3 @@
+; inherits: python
+
+(wildcard (constraint) @regex)
```

Comment:
```suggestion
(wildcard (constraint) @regex)

((directive 
  name: _ @_direc 
  (directive_parameters (string) @bash))
  (#eq? @_direc "shell")
  (#offset! @bash 0 1 0 -1))
```

Could be iffy with wildcards but it's ok for me

Reply:
I had a bash injection in the queries before, but ended up removing it and I no longer favor it. From my personal experience writing snakemake, it's much more useful to have good highlighting of wildcards than of bash, because you're mostly just calling external scripts or short commands with parameters from the rule. More complicated logic is typically outsourced into dedicated files.